example/src/main/java/io/matchmore/sdk/example/MainActivity.kt

```diff
@@ -60,7 +60,7 @@ class MainActivity : AppCompatActivity() {
     private fun MatchMoreSdk.createPollingSubscription() {
```

Comment:
We should also change the MatchMore to Matchmore.

Let's make a consensus: when we go for SDK we go all caps letter.

MatchmoreSDK and Matchmore